src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +70,13 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      */
     public List<People> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+
+        return peopleList.stream()
+                .filter(p -> p.getSex() == People.Sex.WOMEN
+                && p.getAge() >= fromAge && p.getAge() <= femaleToAge
+                || p.getSex() == People.Sex.MAN
+                && p.getAge() >= fromAge && p.getAge() <= maleToAge)
+                .collect(Collectors.toList());
     }
```

Comment:
Anonymous class here too and code may be simplified
```suggestion

                (people -> people.getSex() == People.Sex.MAN
                ? people.getAge() >= fromAge && people.getAge() <= maleToAge
                : people.getAge() >= fromAge && people.getAge() <= femaleToAge)
              
    }
```